WordPress/gutenberg
# Added WhatsApp icon to the Social Icon Block

## What?

I have added the Whatsapp icon to the Social Icons Block, as my first Gutenberg contribuition.

## Why?
WhatsApp is really famous in Brazil, it's used for everything here, so I wanted my first Gutenberg contribution to impact these users.

This is related to: https://github.com/WordPress/gutenberg/issues/30378
 


## How?


## Testing Instructions





1. Open a Post or Page
2. Add a Social Block
3. Add the WhatsApp Icon
4. Add content to it, normally https://wa.me/your-phone-number
5. Save and make sure it appears as expected in the editor as well as the preview


## Screenshots or screencast 

In the editor:

![image](https://user-images.githubusercontent.com/1044309/177210976-ff09c509-8470-4cfa-aa48-b09b51b952f4.png)

Result in the preview:

![image](https://user-images.githubusercontent.com/1044309/177210954-699d386a-3f15-4245-87ca-bbbaf26cfc55.png)
